WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -375,7 +379,9 @@ extension AddAttributeOptionsViewController {
 
         alertController.addCancelActionWithTitle(Localization.cancelAction)
         alertController.addDestructiveActionWithTitle(Localization.removeAction) { [weak self] _ in
-            self?.removeCurrentAttribute()
+            guard let self = self else { return }
+            self.analytics.track(event: WooAnalyticsEvent.Variations.removeAttributeButtonTapped(productID: self.viewModel.product.productID))
+            self.removeCurrentAttribute()
```

Comment:
I noticed that when we remove an attribute, it fires `remove_product_attribute_button_tapped` but also these events:

* `create_product_attribute`
* `create_product_attribute_success`

Is that expected? I see it's because `removeCurrentAttribute()` calls `performProductUpdate()` (which fires the `create_*` events). But in this case, the product update isn't creating anything.

If we want to track the update (and the update success/fail) when the attribute is removed, would it make sense to use a more general name (e.g. `update_product_attribute` instead of `create_product_attribute`)? If we want to keep track of what kind of update is done, we could pass params like `create`/`remove` depending on what the update is for.

Reply:
That's probably a good idea. I'll make that change!